hooks/build

```
@@ -5,4 +5,10 @@
 
 docker build --build-arg VCS_REF=$GIT_SHA1 \
              --build-arg GRAYLOG_VERSION=`cat VERSION` \
+             --file docker/oss/Dockerfile \
              --tag $IMAGE_NAME .
+
+docker build --build-arg VCS_REF=$GIT_SHA1 \
+             --build-arg GRAYLOG_VERSION=`cat VERSION` \
```

Can we add BUILD_DATE here? It's listed in the labels in the Dockerfile, but it isn't populated. 

```suggestion
             --build-arg GRAYLOG_VERSION=`cat VERSION` \
             --build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` \
```